app/lib/hesa/parsers/itt_record.rb

```diff
@@ -15,13 +15,13 @@ class IttRecord
         course_subject_one: "F_SBJCA1",
         course_subject_two: "F_SBJCA2",
         course_subject_three: "F_SBJCA3",
-        itt_start_date: "F_COMDATE",
+        commencement_date: "F_COMDATE",
         itt_end_date: "F_EXPECTEDENDDATE",
         employing_school_urn: "F_SDEMPLOY",
         lead_school_urn: "F_SDLEAD",
         mode: "F_MODE",
         course_age_range: "F_ITTPHSC",
-        trainee_start_date: "F_ITTCOMDATE",
+        itt_commencement_date: "F_ITTCOMDATE",
```

Comment:
Would it be worth naming this something like `hesa_itt_comdate` to more closely match the hesa field, and then later name it something more register-ey when we have done our logic to choose the one we want?

Reply:
Not sure, then everything in this file could be prefixed by `hesa`.. 

We use it like this:

```
def trainee_start_date
  hesa_trainee[:itt_commencement_date].presence || hesa_trainee[:commencement_date]
end
```

so the fact that its on the `hesa_trainee` i think makes it clear?